scripts/runPostCrab.py

```diff
@@ -152,18 +189,19 @@ def main():
     # if there is more than one sample then we're in trouble, crash here
     assert( len(values) == 1 )
     dataset_name, dataset_id, dataset_nevents = values[0]
-    print "True"
+    print "done"
+
+    print("")
     
-    print "##### Get info from crab (logs, outputs, report)"
+    print "##### Get info from crab (outputs, report)"
     # Since the API outputs AND prints the same data, hide whatever is printed on screen
     saved_stdout, saved_stderr = sys.stdout, sys.stderr
     if not options.debug:
         sys.stdout = sys.stderr = open(os.devnull, "w")
     taskdir = os.path.join(workArea, 'crab_' + requestName)
-    # list logs
-    log_files = crabCommand('getlog', '--dump', dir = taskdir )
     # list output
     output_files = crabCommand('getoutput', '--dump', dir = taskdir )
+    output_files_xrootd = crabCommand('getoutput', '--dump', '--xrootd', dir = taskdir )
```

Comment:
Mainly to be site independent. You don't have to know where the file is stored anymore. I first tried with the default ``pfn``, but root does not handle ``srm://`` url.